### PR TITLE
Add UTF8 Unit Test

### PR DESCRIPTION
**Pull request by neelanshsahai**
### Short description
I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

File: test/org/loklak/tools/storage/UTF8Test.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
+package org.loklak.tools.storage;
+
+import org.junit.Assert;
```
Comment (simsausaurabh) [inline test/org/loklak/tools/storage/UTF8Test.java]: import ```assertEquals``` and ```assertArrayEquals``` individually and call them explicitly.

Comment (simsausaurabh) [inline test/org/loklak/tools/storage/UTF8Test.java]: import ```assertEquals``` and ```assertArrayEquals``` individually and call them explicitly.

Comment (neelanshsahai) [reply test/org/loklak/tools/storage/UTF8Test.java]: Okay Sir... :+1:

**Comment by simsausaurabh** [comment]
@neelanshsahai any updates?

**Comment by neelanshsahai** [comment]
no

**Comment by simsausaurabh** [comment]
@neelanshsahai Please do the requested changes and update the PR, PR would not break this time, the required changes are made.

**Comment by neelanshsahai** [comment]
ok, will do that

**Comment by mariobehling** [comment]
Closing this due to inactivity.

**Pull request closed without merging**